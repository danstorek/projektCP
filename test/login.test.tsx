import React from 'react'
import Index from '../pages/login'
import renderer from 'react-test-renderer'

describe('Index', () => {
  it('renders the html we want', async () => {
    const component = renderer.create(
        <Index />
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
