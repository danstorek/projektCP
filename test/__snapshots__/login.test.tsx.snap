// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders the html we want 1`] = `
<div
  className="container"
>
  <main
    className="mainwhite"
  >
    <button
      className="buttontoggle"
      onClick={[Function]}
    >
      Dark Mode
    </button>
    <h1
      className="title"
    >
      Login
    </h1>
    <br />
    <form
      action="#"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <h2
        className="title2"
      >
        Email
      </h2>
      <input
        disabled={false}
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        type="email"
        value=""
      />
      <h2
        className="title2"
      >
        Password
      </h2>
      <input
        disabled={false}
        name="password"
        onBlur={[Function]}
        onChange={[Function]}
        type="password"
        value=""
      />
      <br />
      <button
        className="btn btn-primary"
        disabled={false}
        type="submit"
      >
        Login
      </button>
      <a
        href="/register"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Register
      </a>
    </form>
    <p
      className="ok"
    >
      
    </p>
    <a
      href="/main"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Click here to get to the main page.
    </a>
  </main>
</div>
`;
